features/developer_portal/admin/account/invoices.feature

```diff
@@ -14,13 +22,13 @@ Feature: Dev Portal Buyer Invoices
     And 1 month pass
     When the buyer logs in to the provider
     And the buyer is reviewing their account settings
-    Then they should be able to see an invoice for last month
+    Then they should have access last's month invoice
```

Comment:
```suggestion
    Then they should see last month's invoice
```

Actually.... this is weird! This step - "the buyer logs in to the provider" - what??
The buyer is not supposed to log in to the admin portal :upside_down_face: 

I thought this is for providers (tenants) to see the invoices that were issued for them by **master**. But now I'm not sure!

Reply:
I agree 💯, it is super confusing. The term _tenant_ is not used in the cucumbers, instead they decided to use "buyer" for every "account". Technically a **tenant** is a master provider's buyer and a **developer** is a provider's buyer account, correct?

Origin is from `3scale/system` 8 years ago and probably even older: https://github.com/3scale/system/pull/7015/files#diff-f363de940c8d972611ab8d052715bf5585211d646bce7788bc5bc27f276e0e57R270-R289

When we clean up features/step_definitions/session_steps.rb we'll probably need to have a discussion about what is the correct term for each case.

Reply:
Yeah, actually, I realized what that step was supposed to mean later, in https://github.com/3scale/porta/pull/3663#discussion_r1482966168